python/llm/dev/benchmark/LongBench/README.md

```diff
@@ -0,0 +1,93 @@
+# LongBench Benchmark Test
+
+LongBench Benchmark allows users to test LongBench benchmark and record them in some json files. Users can provide models and related information in `config.yaml` and `config` directory.
+
+Before running, make sure to have [ipex-llm](../../../../../README.md) installed.
+
+## Dependencies
+
+```bash
+pip install omegaconf
+pip install datasets
+pip install jieba
+pip install fuzzywuzzy
+pip install rouge
+```
+
```

Comment:
Add Load Data section
You can download and load the LongBench data through the Hugging Face datasets ([🤗 HF Repo](https://huggingface.co/datasets/THUDM/LongBench)):

```
from datasets import load_dataset

datasets = ["narrativeqa", "qasper", "multifieldqa_en", "multifieldqa_zh", "hotpotqa", "2wikimqa", "musique", \
            "dureader", "gov_report", "qmsum", "multi_news", "vcsum", "trec", "triviaqa", "samsum", "lsht", \
            "passage_count", "passage_retrieval_en", "passage_retrieval_zh", "lcc", "repobench-p"]

for dataset in datasets:
    data = load_dataset('THUDM/LongBench', dataset, split='test')
    data = load_dataset('THUDM/LongBench', f"{dataset}_e", split='test')
```